src/query/sql/src/planner/binder/copy_into_table.rs

```diff
@@ -215,6 +220,7 @@ impl<'a> Binder {
             self.bind_copy_from_query_into_table(bind_ctx, plan, &select_list, &None)
                 .await
         } else {
+            // TODO seems we need to collect_files here
```

Comment:
Will this be completed in a subsequent PR? If this PR is merged, will collect_files be missed?

Reply:
oh, forget to remove this TODO.

please see `bind_copy_into_table` and `bind_copy_from_attachment` (which call this method)